starter/src/main/java/com/starter/fullstack/rest/InventoryController.java

```diff
@@ -25,6 +27,16 @@ public InventoryController(InventoryDAO inventoryDAO) {
     this.inventoryDAO = inventoryDAO;
   }
 
+  /**
+   * Save Inventory.
+   * @param inventory Inventory to Save/Update.
+   * @return Created/Updated Inventory.
+   */
+  @PostMapping
+  public Inventory create(@RequestBody Inventory inventory) {
```

Comment:
How can we check if the data that you send to the controller method is valid or not?

Reply:
You can use the @Valid annotation next to where @RequestBody is to ensure validation of the whole object.

Reply:
Added the valid tag, my create test was failing at first. Turns out Product Type can't be null for the inventory to be valid.